controllers/test/xjoinpipeline_controller_test.go

```diff
@@ -2,6 +2,10 @@ package test
 
 import (
 	"fmt"
+	"reflect"
```

Comment:
Why were these three lines (reflect, strconv, time) added to the import in the same PR as the wording update?